pkgs/top-level/python-packages.nix

```diff
@@ -1088,7 +1088,7 @@ self: super: with self; {
 
   awswrangler = callPackage ../development/python-modules/awswrangler { };
 
-  ax = callPackage ../development/python-modules/ax { };
+  ax-platform = callPackage ../development/python-modules/ax-platform { };
```

Comment:
Unless you are going to package the "ax" now, can we get an alias?

Reply:
I don't think that's a good idea. What if we want to package ax in the future?

Reply:
Sure, my only concern was that if ax package might as well use some visibility before that other packaging happens.